TCAT/InformationViewController.swift

```diff
@@ -0,0 +1,125 @@
+        titleLabel.text = "Made by AppDev"
+        titleLabel.backgroundColor = .clear
+        titleLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(tcatImage.snp.bottom).offset(44)
+            make.centerX.equalToSuperview()
+            make.bottom.equalTo(descriptionLabel.snp.top).offset(-12)
+            make.height.equalTo(19)
+        }
+        
+        descriptionLabel.font = UIFont(name: FontNames.SanFrancisco.Regular, size: 14)
+        descriptionLabel.textColor = UIColor.primaryTextColor
+        descriptionLabel.text = "Cornell University\nApp Development Project Team"
+        descriptionLabel.numberOfLines = 0
+        descriptionLabel.backgroundColor = .clear
+        descriptionLabel.textAlignment = .center
+        descriptionLabel.snp.makeConstraints { (make) in
+            make.height.equalTo(34)
+//            make.bottom.equalTo(view.snp.centerY).offset(-10)
+            make.centerX.equalToSuperview()
+        }
+        
+        sendFeedbackButton.addTarget(self, action: #selector(openBugReportForm), for: .touchUpInside)
+        sendFeedbackButton.setTitle("Send Feedback", for: .normal)
+        sendFeedbackButton.setTitleColor(UIColor.tcatBlueColor, for: .normal)
+        sendFeedbackButton.titleLabel?.font = UIFont(name: FontNames.SanFrancisco.Regular, size: 14)
+        sendFeedbackButton.backgroundColor = UIColor.white
+        sendFeedbackButton.layer.borderColor = UIColor.lineDarkColor.cgColor
+        sendFeedbackButton.layer.borderWidth = 0.5
+        sendFeedbackButton.snp.makeConstraints { (make) in
+            make.centerX.equalToSuperview()
+            make.leading.equalToSuperview().offset(-0.5)
+            make.trailing.equalToSuperview().offset(0.5)
+            make.height.equalTo(44)
+            make.top.equalTo(descriptionLabel.snp.bottom).offset(37)
+        }
+        
+        visitWebsiteButton.addTarget(self, action: #selector(openTeamWebsite), for: .touchUpInside)
+        visitWebsiteButton.setTitle("Visit Our Website", for: .normal)
+        visitWebsiteButton.setTitleColor(UIColor.primaryTextColor, for: .normal)
+        visitWebsiteButton.titleLabel?.font = UIFont(name: FontNames.SanFrancisco.Regular, size: 14)
+        visitWebsiteButton.backgroundColor = .white
+        visitWebsiteButton.layer.borderColor = UIColor.lineDarkColor.cgColor
+        visitWebsiteButton.layer.borderWidth = 0.5
+        visitWebsiteButton.snp.makeConstraints { (make) in
+            make.centerX.equalToSuperview()
+            make.leading.equalTo(sendFeedbackButton)
+            make.trailing.equalTo(sendFeedbackButton)
+            make.height.equalTo(44)
+            make.top.equalTo(sendFeedbackButton.snp.bottom).offset(-0.5)
+        }
+    }
+    
+    @objc func backButtonClicked() {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    @objc func openBugReportForm() {
+        let betaFormURL = "https://goo.gl/forms/u2shinl8ddNyFuZ23"
```

Comment:
You have the link in two places. Maybe it should be a constant somewhere else so we can update it just in one place

Reply:
Not sure what you mean by this

Reply:
Oh my bad, I thought for some reason you had the URL in two places